pkg/assets/builder.go

```diff
@@ -151,7 +151,11 @@ func (a *AssetBuilder) RemapImage(image string) (string, error) {
 		normalized := image
 
 		// Remove the 'standard' kubernetes image prefix, just for sanity
-		normalized = strings.TrimPrefix(normalized, "k8s.gcr.io/")
+		if !util.IsKubernetesGTE("1.10", a.KubernetesVersion) && strings.HasPrefix(normalized, "gcr.io/google_containers/") {
+			normalized = strings.TrimPrefix(normalized, "gcr.io/google_containers/")
```

Comment:
We probably could just always remove the prefix, but what you've done is safer :-)

Reply:
Well, first commit and all, I'm a bit more defensive than I'd be on a well-known code base ;)